### PR TITLE
improve JOY token tooltip info

### LINKED ISSUE #6127: JOY purchasing tooltip update
## Context
Drive more JOY purchases from Gleev app users. Current tooltip only shows in the studio, and does not for the users on the viewer side. 

Screenshot 2024-04-08 at 14 02 16

"JOY token is a native crypto asset of Joystream blockchain. It is used for platform governance, purchasing NFTs, trading creator tokens, and covering blockchain processing fees. They are listed on [MEXC](https://www.mexc.com/exchange/JOYSTREAM_USDT) exchange under JOYSTREAM ticker."

## Scope
1. Update tooltip wording.
`JOY token is a native crypto asset of Joystream blockchain which powers Gleev. It is used for  trading Creator Tokens, NFTs and covering blockchain processing fees. It is also used for voting on proposals and partaking in council elections. [Purchase JOY](https://www.joystream.org/token/).`
2. Add tooltip to viewer side as well.

### PR DESCRIPTION
**Pull request by kdembler**
Resolves #6127

### CODE REVIEW

File: packages/atlas/src/components/JoyTokenIcon/JoyTokenIcon.tsx [hunk 1]
```diff
@@ -76,13 +76,12 @@ export const JoyTokenIcon: FC<JoyTokenIconProps> = ({
...
           <TooltipText as="span" variant="t100">
-            {atlasConfig.joystream.tokenTicker} token is a native crypto asset of Joystream blockchain. It is used for
-            platform governance, purchasing NFTs, trading creator tokens, and covering blockchain processing fees. They
-            are listed on{' '}
-            <Anchor href="https://www.mexc.com/exchange/JOYSTREAM_USDT" target="__blank">
-              MEXC
-            </Anchor>{' '}
-            exchange under JOYSTREAM ticker
+            {atlasConfig.joystream.tokenTicker} token is a native crypto asset of Joystream blockchain which powers
+            Gleev. It is used for trading Creator Tokens, NFTs and covering blockchain processing fees. It is also used
```
Comment (attemka) [inline packages/atlas/src/components/JoyTokenIcon/JoyTokenIcon.tsx]:
```suggestion
            ${atlasConfig.general.appName}. It is used for trading Creator Tokens, NFTs and covering blockchain processing fees. It is also used
```

File: packages/atlas/src/components/JoyTokenIcon/JoyTokenIcon.tsx [hunk 2]
```diff
@@ -76,13 +76,12 @@ export const JoyTokenIcon: FC<JoyTokenIconProps> = ({
...
+            for voting on proposals and partaking in council elections.{' '}
+            <ColorAnchor href="https://www.joystream.org/token/" target="_blank">
+              Purchase JOY
```
Comment (attemka) [inline packages/atlas/src/components/JoyTokenIcon/JoyTokenIcon.tsx]:
```suggestion
              Purchase ${atlasConfig.joystream.tokenTicker}
```

File: packages/atlas/src/views/global/YppLandingView/sections/YppRewardSection.tsx [hunk 3]
```diff
@@ -122,13 +122,12 @@ export const YppRewardSection: FC = () => {
...
                 <TooltipText as="span" variant="t100">
-                  {atlasConfig.joystream.tokenTicker} token is a native crypto asset of Joystream blockchain. It is used
-                  for platform governance, purchasing NFTs, trading creator tokens, and covering blockchain processing
-                  fees. They are listed on{' '}
-                  <ColorAnchor href="https://www.mexc.com/exchange/JOYSTREAM_USDT" target="__blank">
-                    MEXC
-                  </ColorAnchor>{' '}
-                  exchange under "JOYSTREAM" ticker.
+                  {atlasConfig.joystream.tokenTicker} token is a native crypto asset of Joystream blockchain which
+                  powers Gleev. It is used for trading Creator Tokens, NFTs and covering blockchain processing fees. It
```
Comment (attemka) [inline packages/atlas/src/views/global/YppLandingView/sections/YppRewardSection.tsx]: same as 2 above

**Review by attemka — CHANGES_REQUESTED** [review]

**Comment by WRadoslaw** [comment]
Could we instead of pointing to the Joystream website just redirect the user to the Portfolio page and show ChangeNow modal?

cc @dmtrjsg

**Comment by kdembler** [comment]
@WRadoslaw Joystream website gives more purchasing options and lists exchanges, I think it's a better choice

**Review by attemka — APPROVED** [review]

**Pull request merged**